lib/Swagger/src/ContentEntity/Models/Content.php

```diff
@@ -69,8 +71,11 @@ class Content implements ArrayAccess
         'site_name' => 'string',
         'content_type' => 'string',
         'modified' => 'string',
+        'authors' => '\Swagger\Client\ContentEntity\Models\Author[]',
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "authors" 10 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)